install/Docker/dockerfiles/Dockerfile.etcd

```diff
@@ -0,0 +1,65 @@
+#(C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+#Licensed under the Apache License, Version 2.0 (the "License"); you may
+#not use this file except in compliance with the License. You may obtain
+#a copy of the License at
+#
+#    http:#www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#License for the specific language governing permissions and limitations
+# under the License.
+
+# This is Dockerfile to build etcd image
+FROM ubuntu:18.04 as etcd-build
+
+LABEL version="1.0"
+LABEL ubuntuVersion="18.04"
+LABEL etcdVersion="3.4.0"
+LABEL description="etcd docker image built on top of ubuntu:18.04"
+
+WORKDIR /etcd/
+
+# userid and groupid of odimra user and odimra group respectively
+# which will present be on the host machine.
+ARG ODIMRA_USER_ID
+ARG ODIMRA_GROUP_ID
+
+RUN apt-get update -q=3 && apt-get -q install -q=3 -y \
+        vim \
```

Comment:
Please remove vim package it's not required and it increases the image size.

Reply:
@bharath-b-hpe Addressed